cmake/macros.cmake

```diff
@@ -29,7 +29,7 @@ ENDMACRO()
 
 # Message Output
 macro(MESSAGE_WARN _TEXT)
-  message(STATUS "*** WARNING: ${_TEXT}")
+  message(WARNING "*** WARNING: ${_TEXT}")
```

Comment:
CMake 3.0 does not have `WARNING`: https://cmake.org/cmake/help/v3.0/command/message.html

Reply:
CMake 3.0 does support message WARNING.

Reply:
Oops, I confused it with `message(DEBUG)`.